polar2grid/core/rescale_configs/rescale.ini

```diff
@@ -180,6 +180,13 @@ method=linear
 min_in=0.0
 max_in=50.0
 
+[rescale:default_sfr]
+data_kind=sfr
```

Comment:
I think this needs to be "snow_fall_rate". Otherwise you could add `product_name: sfr` which would accomplish the same thing in this case. What we want is to configure this to match any product where `data_kind` equals `snow_fall_rate` (as configured in the reader) or `product_name` equals `srf` (as named in the reader). You don't need both.

Reply:
I tried that, and the log file implies it is doing what it should:
```
[2019-03-27 19:27:18,383] : PID  30408 : DEBUG    : polar2grid.core.rescale : get_rescale_options : Product sfr found in rescale config: {'satellite': 'noaa19', 'units': 'mm/hr', 'data_type': 'uint1', 'min': '0.0', 'reader': 'mirs', 'product_name': 'sfr', 'instrument': 'amsua-mhs', 'grid_name': '203_10km', 'max_in': 5.0, 'inc_by_one': False, 'data_kind': 'snow_fall_rate', 'method': 'linear'}
```

But it still doesn't look correct.  I will take to Carl and see what he says.

Reply:
@spruceboy was AWIPS updated to the new 0-5 range?

Reply:
Hmm, good point, let me check with Carl.

Reply:
Hmm, #2 - looking at the raw netcdf output of polar2grid, it looks like most of the land is masked off with the value of "6", and no snow is masked off with a value of 1.  
Is there some sort of special value in this product for invalid values? 

Here is the data I am looking at:
http://hippy.gina.alaska.edu/distro/tmp/waffle/

Reply:
This is what the netcdf source data looks like.  The blue is I think some sort of invalid data.. 
Is this handled somehow in the scmi writer so we are not noticing it? 

![Screen Shot 2019-03-27 at 12 02 48 PM](https://user-images.githubusercontent.com/243433/55109396-9d85a080-508a-11e9-9a72-06107178068e.png)

Reply:
@spruceboy Could you please push your changes to rescale.ini so I can see/use them.

Reply:
It looks like those blue values are 0.0  - perhaps the SCMI writer is treating those as nodata, which would hid this problem.

Reply:
Also it looks like you probably have some custom changes to `awips_backend.ini`. What does that look like?